release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1254,6 +1254,10 @@ If you are using the option `--keep-manifest-list=true`, the only valid value fo
 
 * With Serverless 0.10, the latest supported versions were updated for `trigger`, `subscription`, `channel`, and `IMC`.  Static models corresponding to each showed an API version of `beta`. The API version for eventing resources is now updated to `v1` and the UI now shows the latest supported version. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1890104[*BZ#1890104*])
 
+* Previously, the API server could fail to create a resource and return a 409 status code due to a conflict while updating a resource quota resource. This issue has now been fixed and the OpenShift Console web app will attempt to retry the request while receiving a 409 status code. At most three attempts will be made which is often sufficient to complete the request. In the event that a 409 continues to occur, an error will be displayed in the console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1928228[*BZ#1928228*])
+
+* Previously, while creating a sample application the Developer Console created multiple resources. These resources depend on each other and should be created in a specific order. Due to this, an admission plugin fails to check one of these resources and the sample application could not be created. This issue has been fixed now, and instead of triggering all resources at the same time the code creates now all resources step by step, thus making a sample application more stable. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1933665[*BZ#1933665*])
```

Comment:
* While producing a sample app, the Developer Console creates multiple resources that depend on each other and must be completed in a specific order. Previously, an admission plugin sometimes could not check one of these resources, preventing the Developer Console from generating the Sample app. This issue has been fixed. The code creates the resources in the required order, so producing a sample app is more stable.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1933665[*BZ#1933665*])